Silicon/TigerlakePchPkg/Include/PchAccess.h

```diff
@@ -0,0 +1,81 @@
+/** @file
+  Macros that simplify accessing PCH devices's PCI registers.
+
+  Copyright (c) 1999 - 2016, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+#ifndef _PCH_ACCESS_H_
+#define _PCH_ACCESS_H_
+
+#include "PchLimits.h"
+#include "PchReservedResources.h"
+#include <GpioPinsVer2H.h>
+#include <GpioPinsVer2Lp.h>
+
+#define PCH_TGL                 1
+#define PCH_ICL                 0
+#define HPET_BASE_ADDRESS       0xFED00000
+
+
+#ifndef SLE_FLAG
+#ifndef STALL_ONE_MICRO_SECOND
+#define STALL_ONE_MICRO_SECOND 1
+#endif
+#ifndef STALL_ONE_SECOND
+#define STALL_ONE_SECOND 1000000
+#endif
+#else  // SLE_FLAG
+#ifndef STALL_ONE_MICRO_SECOND
+#define STALL_ONE_MICRO_SECOND 0
+#endif
+#ifndef STALL_ONE_SECOND
+#define STALL_ONE_SECOND 0
+#endif
+#endif // SLE_FLAG

```

Comment:
SLE_FLAG is not defined, we could remove related defination.

Reply:
Done removed.